components/generic/PaginationNav.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <b-pagination-nav
```

Comment:
The links for next/previous page seem to change the page number in the URL incorrectly.